server/server.go

```diff
@@ -44,6 +49,7 @@ type SignalHandler interface {
 // Config for a Server
 type Config struct {
 	MetricsNamespace  string `yaml:"-"`
+	Network           string `yaml:"network"`
```

Comment:
For consistency this would be `http_listen_network`

Reply:
ok, I fix it